scout/adapter/mongo/variant.py

```diff
@@ -703,29 +705,48 @@ def hgnc_overlapping(self, variant_obj: dict, limit: int = None) -> Iterable[Dic
         limit: A maximum count of returned variants is introduced: mainly this is a problem when SVs are huge since there can be many genes and overlapping variants.
                We sort to offer the LIMIT most severe overlapping variants.
         """
-        category = (
-            {"$in": ["sv", "mei"]}
-            if variant_obj["category"] == "snv"
-            else {"$in": ["sv", "snv", "mei", "cancer", "cancer_sv"]}
-        )
 
-        variant_type = variant_obj.get("variant_type", "clinical")
-        hgnc_ids = variant_obj["hgnc_ids"]
-        if not limit:
-            limit = 30 if variant_obj["category"] == "snv" else 45
+        def dna_overlapping(
+            hgnc_ids: List[int], variant_type: str, limit: Optional[int]
+        ) -> Iterable[Dict]:
+            """Return DNA other categories of DNA variants matching the genes of the DNA variant in question."""
+            category = (
+                {"$in": ["sv", "mei"]}
+                if variant_obj["category"] == "snv"
+                else {"$in": ["sv", "snv", "mei", "cancer", "cancer_sv"]}
+            )
 
-        query = {
-            "$and": [
-                {"case_id": variant_obj["case_id"]},
-                {"category": category},
-                {"variant_type": variant_type},
-                {"hgnc_ids": {"$in": hgnc_ids}},
-                {"variant_id": {"$ne": variant_obj["variant_id"]}},
-            ]
-        }
-        sort_key = [("rank_score", pymongo.DESCENDING)]
+            if not limit:
+                limit = 30 if variant_obj["category"] == "snv" else 45
 
-        return self.variant_collection.find(query).sort(sort_key).limit(limit)
+            query = {
+                "$and": [
+                    {"case_id": variant_obj["case_id"]},
+                    {"category": category},
+                    {"variant_type": variant_type},
+                    {"hgnc_ids": {"$in": hgnc_ids}},
+                    {"variant_id": {"$ne": variant_obj["variant_id"]}},
+                ]
+            }
+            sort_key = [("rank_score", pymongo.DESCENDING)]
+            return self.variant_collection.find(query).sort(sort_key).limit(limit)
+
+        def wts_overlapping(hgnc_ids: List[int], variant_type: str) -> Iterable[Dict]:
```

Comment:
I would suggest replacing wts with outlier. Not that we have any others at the moment, but this does return all omics variants. The docstring is perfect! 😁

Reply:
Sure. I'll rename it all over the place

Reply:
Or just here, for simplicity sake